XenModel/Messages.resx

```diff
@@ -2167,7 +2167,8 @@ Deleting this bond will disrupt traffic through the secondary interface on the b
     <value>It is not possible to activate this virtual disk on VM '{0}' as the VM is not running.</value>
   </data>
   <data name="CANNOT_ACTIVATE_VD_VM_NEEDS_TOOLS" xml:space="preserve">
-    <value>You must install XenServer Tools on VM '{0}' before you can activate this virtual disk for the VM.</value>
+    <value>You must install XenServer Tools or the I/O drivers on VM '{0}' before you can activate this virtual disk for the VM.</value>

```

Comment:
This is a bit ugly, but it's probably the best solution. But let's change it to "XenServer Tools or I/O drivers" (remove "the").